### ISSUE TITLE
Crashes when pressing `Shift_R+F9` after latest update

### ISSUE THREAD

**Issue by matteo-pacini** [flathub/org.freedesktop.Platform.VulkanLayer.MangoHud#44]
Hi there,

updated MangoHud this morning to this version:

```
00bbf0a5c505cd0b7877f54564a292b03ab1484ec1f259f51284eb72ee0391f4
Revert "build: Do not link statically with libstdc++" 
```

Tested a few games after that, and they either crash or get stuck when I try to cycle through the FPS limiting options (tested Age Of Wonders 4 and Helldivers 2)

Here's my MangoHud config for reference:

```
fps_limit=0,144,120,90,60
show_fps_limit
toggle_fps_limit=Shift_R+F9
```

Reverting to the previous version fixes the issue:

```
flatpak --user update \
--commit=cb7cb4464e8c2da28ec517b4d53eefa6d86fdf1fd1bb76eda5a85a5f29b43050 \
org.freedesktop.Platform.VulkanLayer.MangoHud/x86_64/23.08
```

**Comment by jntesteves** [comment]
The problem here is not the `fps_limit`, it's the `Shift_R+F9` shortcut, it's already in use for resetting `fps_metrics`. I can reproduce, there's currently a bug that's causing a crash when pressing `Shift_R+F9` while not using `fps_metrics`. I've reported it to upstream at flightlessmango/MangoHud#1257

**Comment by jntesteves** [comment]
Fixed in #46. @matteo-pacini can you test if the build on that PR solves the issue for you too? Thanks.

**Comment by jntesteves** [comment]
A fixed build was published on Flathub overnight, you'll get it on next update. Feel free to re-open this issue if you still have problems. Thanks.
